iocage/cli/start.py

```diff
@@ -50,7 +50,9 @@ def cli(rc, jails, log_level):
         logger.log(f"Starting jail '{jail.humanreadable_name}'")
         try:
             jail.start()
-        except:
+        except Exception as e:
+            raise
+            logger.error(str(e))
             exit(1)
```

Comment:
The commit title and the content don't seem to make sense. Your log will never be reached (`raise` is before it) and then you exit on 1.